src/misc/lv_bidi.c

```diff
@@ -364,7 +370,11 @@ static bool lv_bidi_letter_is_rtl(uint32_t letter)
 static bool lv_bidi_letter_is_neutral(uint32_t letter)
 {
     uint16_t i;
-    static const char neutrals[] = " \t\n\r.,:;'\"`!?%/\\-=()[]{}<>@#&$|";
+    const char * neutrals = " \t\n\r.,:;'\"`!?%/\\-=()[]{}<>@#&$|";
+    if(custom_neutrals) {
+        neutrals = custom_neutrals;
+    }
+
```

Comment:
This way we cannot easily revert to the default.  I suggest
```suggestion
    const char * default_neutrals = " \t\n\r.,:;'\"`!?%/\\-=()[]{}<>@#&$|";
    if(custom_neutrals) {
        neutrals = custom_neutrals;
    } else  {
       neutrals =  default_neutrals;
    }

```

Reply:
The current method is that as long as custom_neutrals is set to NULL, the default neutrals will be used.

Reply:
In Line 375 the original `neutrals` are overwritten with `custom_neutrals`, so there is no way to get back the default settings anymore.

Reply:
> neutrals

I don't quite understand, neutrals is no longer a static pointer, why there is no way to get back the default settings anymore?

Reply:
I was stupid, sorry :smile: Somehow I thought it's  `static const char * neutrals = ...`.